### ISSUE TITLE
3 Test and evaluate model

### ISSUE THREAD

**Issue by ianbunag** [ianbunag/cmpe-255-data-mining-project#50]
# Description

Same with training and validation, the training and test sets will be scaled on a scaler fit with the training set. Use the last fold for this stage.

# Stage
Evaluation

# Recommended visual output

Dataframe of test results containing actual and predicted RSV_TP_LEVEL values.

Accuracy, precision, and recall metrics.

Consider including ROC/AUC analysis 

# Technical details

## Useful python functions
- `numpy.round` and `numpy.clip` to constrain predicted values to RSV_TP_LEVEL to categorical range
